### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by wylie**
Created a mixin for our buttons and a loop to loop through them. Cleans up the page a little and removes a lot of repetition of CSS.

Also cleaned up two files.

### CODE REVIEW

File: less/_button.less [hunk 1]
```diff
@@ -121,6 +121,43 @@
 // 
 // Styleguide 2.9
 
+.btn-kind(@kind) {
```
Comment (wylie) [inline less/_button.less:124]: For some reason this wouldn't work when I put it in the _mixins.less file, so it lives here

File: less/_button.less [hunk 2]
```diff
@@ -155,110 +192,16 @@
...
-		&:active,
-		&.active {
-			background-color: @btn-disabled-bgcolor-active;
-			border-color: @btn-disabled-bordercolor-active;
-		}
-
-		&:focus {
-			outline: none;
-		}
+	// loop through the button kinds and hit the .btn-kind mixin
```
Comment (wylie) [inline less/_button.less:195]: Here is our new loop

File: less/_button.less [hunk 3]
```diff
@@ -155,110 +192,16 @@
 		background-image: none !important;
 	}
 
-	&.search {
```
Comment (wylie) [inline less/_button.less:158]: 102 lines of CSS now 43 lines of CSS, boom!

**Comment by amyh** [comment]
nice work!

**Pull request merged**